azure/services/scalesets/scalesets.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 	"github.com/go-logr/logr"
 	"github.com/pkg/errors"
+
 	"sigs.k8s.io/cluster-api-provider-azure/util/generators"
 
```

Comment:
nit: is this vertical space intentional

Reply:
Not really, goimports added it for me. I'll remove it.